yarn-project/end-to-end/package.json

```diff
@@ -33,8 +33,7 @@
     "@aztec/l1-artifacts": "workspace:^",
     "@aztec/noir-contracts": "workspace:^",
     "@aztec/p2p": "workspace:^",
-    "@aztec/sequencer-client": "workspace:^",
-    "@aztec/world-state": "workspace:^",
```

Comment:
I believe these were here so they would get automatically listed as `dependencies` in the build manifest when running `yarn prepare`, so the e2e tests are run whenever any of these change.